### ISSUE TITLE
calibre-web flagged as 'misleading' by google

### ISSUE THREAD

**Issue by bartart3d** [janeczku/calibre-web#2870]
My domain is now flagged as being misleading, while I'm pretty sure it is not. 

Using google's search console I found that google thinks there are a few misleading pages, and these all lead to calibre-web (here fictionalised):

http://domainname/calibreweb
https://domainname/calibreweb
https://domainname/calibreweb/
https://domainname/calibreweb/login?next=/calibreweb/

I now added a robot.txt file, disallowing everything. Hopefully this will do something.

My question now is, what can I do about it? Or am I forced to stop using calibre-web on my domain to stop it from being flagged as misleading?

**Additional context**
the domain is a duckdns-domain, which can cause issues sometimes, but here all leads to the /calibreweb location.

**Comment by GoGoSoSlow** [comment]
It's duckdns.  For some reason google is trigger happy with duckdns domains.  I have gotten the same warning with a completely different program.  I reported it here: https://safebrowsing.google.com/safebrowsing/report_error/?hl=en and the warning went away after a week or so.

**Comment by bartart3d** [comment]
I managed to get my domain being available again, after two reports. Only after adding a robots.txt files to my web root saying not to index my site and validating the site as mine in google search console, it got fixed.

But there is always a but. 
Now google search console is inspecting the url every so many times and problems keep popping up. I had to re-validate the site a few days back because it got flagged again. 
I got it fixed again, but now I get an email by the google search console that my site contains content for 'phishing' and 'social engineering'.

I'm running different applications on the site, and it's only the url leading to calibre-web it is listing, so again Google insists on fixing the problem only in the calibre-web application. For the third time now I've sent a request to re-evaluate the site because of calibre-web. 

When it happens once because of using duckdns, that I can believe, twice already is unlikely, but for the third time and always only listing the url to calibre-web... 

I'm turning it off for now and I would suggest re-checking the code for security issues.
